arch/arm/boot/dts/bcm2708-rpi-zero-w.dts

```diff
@@ -4,6 +4,7 @@
 #include "bcm2708-rpi.dtsi"
 #include "bcm283x-rpi-csi1-2lane.dtsi"
 #include "bcm283x-rpi-i2c0mux_0_28.dtsi"
+#include "bcm27xx-rpi-bt.dtsi"
 
 / {
```

Comment:
Oops, missed this one because I was testing build on arm64.

Do you have ideas how to deal with miniuart-bt-overlay best? It also refers to `&expgpio` after my change, and it should work with all Pi models. Is there a way for an overlay to pick one of `&expgpio` or `&gpio`, depending on the main tree? Or any way to add a label alias to point `expgpio` to `&gpio` in the Pi0W tree?

Reply:
The logical way would be to give uart1 a disabled bluetooth node in the base DTBs, just as we did with uart0.

Reply:
Thanks, that sounds like a solution. I'll update the patch shortly.